content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -504,6 +504,33 @@ stops adjusting the target (and sets the `ScalingActive` Condition on itself
 to `false`) until you reactivate it by manually adjusting the target's desired
 replica count or HPA's minimum replica count.
 
+### Migrating Deployments and StatefulSets to horizontal autoscaling
+
+When an HPA is enabled, it is recommended that the value of `spec.replicas` of
+the Deployment and/or Statefulset object is removed.  Without doing so, anytime
```

Comment:
Removed from..?

- source code
- the API
?

We should clarify what removal we're advocating.

Reply:
Agreed:

```suggestion
the Deployment and/or Statefulset object is removed from your configuration file.  Without doing so, anytime
```

Reply:
I'd tweak that for wording:
```suggestion
the Deployment and / or StatefulSet is removed from their
{{< glossary_tooltip text="manifest(s)" term_id="manifest" >}}.  Without doing so, any time
```